lib/metadata/bucketfile/backend.js

```diff
@@ -225,6 +228,9 @@ class BucketFileInterface {
     }
 
     deleteObject(bucketName, objName, params, log, cb) {
+        Object.assign(params, {
```

Comment:
it's used for the placeholder delete versions (when repairing master after deleting a version)